apps/sailfish/qml/pages/KodiPage.qml

```diff
@@ -143,18 +153,21 @@ Page {
 
     function populateKodiMenu() {
         kodiMenuModel.clear();
-        kodiMenuModel.append(kodiMenuModelTemplate.get(0));
+        if (kodi.profiles().count > 1) {
```

Comment:
I'm actually thinking this shouldn't work like the others, in a sense that we always have just this one because this is not something we navigate through the stack. What do you say?